ruby/tests/basic.rb

```diff
@@ -473,7 +473,9 @@ def test_map_basic
       m2 = m.dup
       assert_equal m, m2
       assert m.hash != 0
-      assert_equal m.hash, m2.hash
+      # TODO(teboring): Disable jruby test temperarily for it randomly fails.
```

Comment:
This test case is ran under both regular ruby and jruby. You are disabling both here. Could you just ignore the jruby test?